README.md

```diff
@@ -1,2 +1,110 @@
-# scran
-Command line interface for scran package. Low-level analyses of single-cell RNA-seq data.
+# Wrapper scripts for components of the scran package
+
+Scran implements functions for low-level analyses of single-cell RNA-seq data. Methods are provided for normalization of cell-specific biases, assignment of cell cycle phase, detection of highly variable and significantly correlated genes, identification of marker genes, and other common tasks in routine single-cell analysis workflows.
+
+A vignette with the usage of scran can be found [here](https://bioconductor.org/packages/release/bioc/vignettes/scran/inst/doc/scran.html) 
+
+**Note**: this scripts are for Scran version 1.12. 
+
+## Install
```

Comment:
@rubenchazarra,  you can add some notes on how to install the package even though it's not on Bioconda yet - something like `conda install scran-cli`. Also good to mention that it is better to install into a clean environment